idaes/models/properties/general_helmholtz/components/registry.py

```diff
@@ -19,9 +19,10 @@
 
```

Comment:
Are you able to remove the pylint directive above at this point? From a quick look it appears that you've added doc strings to everything.